src/data/markdown/translated-guides/en/02 Using k6/00 Tutorial/300 Analyze results.md

```diff
@@ -0,0 +1,272 @@
+   import http from "k6/http";
+    
+   export default function () {
+     const url = "https://test-api.k6.io";
+     const payload = JSON.stringify({
+       username: "test_case",
+       password: "1234",
+     });
+   
+     const params = {
+       headers: {
+         "Content-Type": "application/json",
+       },
+       //apply tags
+       tags: {
+         function: "login",
+       },
+     };
+   
+     //Login with tags
+     http.post(`${url}/auth/basic/login`, payload, params);
+   };
+   
+   ```
+   </CodeGroup>
+    
+Now you can filter the results for this tag:
+
+```bash
+jq '. | select(.data.tags.function == "login")' results.json
+```
+
+## Organize requests in groups
+
+You can also organize tags into [_Groups_](/using-k6/tags-and-groups#groups), functions that apply a `group` tag to all requests within its block.
+Groups are very useful for designing tests as a series of logical transactions or blocks.
+
+
+### Context: a new test for user flows
+
+Results filtering isn't very meaningful in a test that makes one request.
+And the API test script is getting long.
+To simplify, write a new test for the following situation:
+
+> Your development team wants to compare the performance of two user-facing components, the Contacts page and the Coinflip game.
+> Write a script that makes it easy to compare the following 
+> - A GET request to `https://test.k6.io/contacts.php`
+> - A POST request to `https://test.k6.io/flip_coin.php` with the query param `?bet=heads`
+> - Another POST to `https://test.k6.io/flip_coin.php` with the query param `?bet=tails`
+
+
+Can you figure out how to script the requests yourself?
+If not, use the following script.
+
+<Collapsible title="user flow example" isOpen="" tag="">
+
+Since this example simulates a human user rather than an API call, it has sleep between each request.
+
+<CodeGroup labels={["user-flow.js"]} lineNumbers={["true"]} showCopyButton={[true]}>
+
+```javascript
+import http from "k6/http";
+import { check, group, sleep } from "k6";
+
+const baseUrl = "https://test.k6.io";
+
+export default function () {
+  http.get(`${baseUrl}/contacts.php`);
+  sleep(1);
+  http.get(`${baseUrl}/flip_coin.php?bet=heads`);
+  sleep(1);
+  http.get(`${baseUrl}/flip_coin.php?bet=tails`);
+  sleep(1);
+}
+```
+
+</CodeGroup>
+
+</Collapsible>
+
+### Add Group functions
+
+Now wrap the two endpoints in different groups:
+Name one group `User contacts page` and another `Coinflip` game.
+
+<CodeGroup labels={["user-flow.js"]} lineNumbers={[]} showCopyButton={[true]}>
+
+```javascript
+//import necessary modules
+import http from "k6/http";
+import { check, group, sleep } from "k6";
+import { Counter, Rate, Trend } from "k6/metrics";
+
+//set baseURL
+const baseUrl = "https://test.k6.io";
+
+export default function () {
+  // User flow
+
+// Put visits to contact page in one group
+  group("User contacts page", function () {
+    http.get(`${baseUrl}/contacts.php`);
+    sleep(1);
+  });
```

Comment:
Yeah, I realized as I wrote this that a progressive tutorial with tags is pretty hard, because the granularity can make really complex configurations quickly. So I opted for groups, and then I wanted to make it as simple as possible.

However, I guess I went too simple. This adds a request. [fda927e](https://github.com/grafana/k6-docs/pull/1192/commits/fda927eac3b1a1b4586994089a776f0806104f1a)

Another downside of the progressive tutorial: changing one snippet entails changing every snippet that is built upon it.